packages/generator/src/service/package-json.ts

```diff
@@ -29,8 +29,8 @@ export function packageJson(
           url: ''
         },
         scripts: {
-          compile: 'npx tsc',
-          doc: 'npx typedoc',
+          compile: 'yarn tsc',
+          doc: 'yarn typedoc',
```

Comment:
Here again, I think we should stay with npx - If a user generates a TS version of the client and tries to transpile it, whithout yarn it won't work.